### PR TITLE
Refactor Ionic Native component

### PR DESCRIPTION
**Pull request by dotNetkow**

### CODE REVIEW

File: src/pages/enterprise/email-composer.md [hunk 1]
```diff
@@ -3,11 +3,30 @@ title: Email Composer
 template: enterprise-plugin
 version: 0.9.2
 minor: 0.9.X
+capacitorConfig:
```
Comment (dotNetkow) [inline src/pages/enterprise/email-composer.md]: With this new design, we'd need to add this frontmatter onto each plugin page. this does mean that the <native-ent-install> component is no longer here - it's moved back into a Template.

Comment (jcesarmobile) [reply src/pages/enterprise/email-composer.md]: how is this supposed to be generated?

**Comment by jcesarmobile** [comment]
I think we should keep the `<native-ent-install plugin-id="plugin-name" variables=""></native-ent-install>` component, and any additional instructions such as preferences or manual edits to config files should be on the README plugin files.

**Comment by dotNetkow** [comment]
OK - sounds good.

**Pull request closed without merging**